leaf-ui/js/onFunctionsNextState.js

```diff
@@ -66,8 +72,14 @@ function fontUp(pPaper) {
  * @param {*} pPaper 
  */
 function fontDown(pPaper) {
-    var new_font = current_font - 1;
+    var elements = analysis.graph.getElements();
+    for (var i = 0; i < elements.length; i++) {
+        var cellView = elements[i].findView(pPaper);
```

Comment:
This is also true in develop, but font sizes don't change on intentions on some of the saved models, such as scheduler or half the intentions in WME. Perhaps an issue with the saved models rather than this code?

Reply:
When you use the V2 models that problem does show up but using the V3 models all the font sizes do change. There is an issue with the V3 models where when you change the font sizes and then go into analysis mode and back to modeling mode the font sizes for some of the intentions reset to the default size.